docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -111,27 +113,39 @@ The IPAM module achieves this task by keeping track of how foreign clusters have
 
 
 #### Reflection
```

Comment:
I feel that part is a bit misplaced here, I was wondering if it should move to https://doc.liqo.io/concepts/offloading/features/api-reflection/ which seems to me its natural place.

Reply:
This section describes how the IPAM is involved in the reflection of endpoints; it is not supposed to describe the whole reflection of API. Therefore I think it should be here and maybe a link in the API reflection page should point to this Section.